content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,138 @@
+---
+content_type: "reference"
+title: Local Files And Paths Used By The Kubelet
+weight: 42
+---
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} is mostly a stateless
+process running on a Kubernetes {{< glossary_tooltip text="node" term_id="node" >}}.
+This document outlines files that kubelet reads and writes.
+
+The kubelet typically uses the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} as
+the source of truth on what needs to run on the Node, and the
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} to retrieve
+the current state of containers. So long as you provide a _kubeconfig_ (API client configuration)
+to the kubelet, the kubelet does connect to your control plane; otherwise the node operates in
+_standalone mode_.
+
+On Linux the kubelet also relies on reading cgroups and various system files to collect metrics.
```

Comment:
@marosset @jsturtevant anything else to mention here regarding Windows?

Reply:
We read a few values from the registry; I am not sure if that is worth mentioning as its pretty common to do that.  Metrics are collected via system calls and perf counters on Windows.

Reply:
We have a few spots where powershell is used for different volume mount operations

- https://github.com/kubernetes/kubernetes/blob/5c14a5779551f678dbb89e9ce2346b9aa5e81d87/pkg/volume/util/subpath/subpath_windows.go#L79
- https://github.com/kubernetes/kubernetes/blob/5c14a5779551f678dbb89e9ce2346b9aa5e81d87/pkg/volume/util/subpath/subpath_windows.go#L120-L121

Plus what is used by the node log query feature.

Do you think we shuold call that out @jsturtevant ?

Reply:
we added very generic language for now